app/assets/stylesheets/earthworks.css

```diff
@@ -134,26 +134,48 @@
   outline: none;
 }
 
+.search-q .input-group {
+  padding: .5rem
+}
+
 .input-group > .search-autocomplete-wrapper {
   border: none;
 }
 
-.search-query-form, .navbar-search .search-query-form{
+.input-group > .search-autocomplete-wrapper input {
+  padding-top: 0;
+  padding-bottom: 0
+}
+
+.search-query-form, .navbar-search .search-query-form {
   border: 1px solid #ABABA9;
-  padding: .5rem;
+  padding: 8px;
   border-radius: .25rem;
   background-color: white;
+  max-height: 47px;
+  width: 100%;
+  @media (min-width: 576px) {
+    max-width: 520px;
+  }
 }
 
 .navbar-search {
   --bs-light-rgb: var(--stanford-fog-light-rgb);
 }
 
+.navbar-search > .container-fluid {
+  justify-content: center;
+  @media (min-width: 768px) {
+    justify-content: space-between;
+  }
+}
 .search-query-form input:focus {
   box-shadow: none;
 }
 .search-query-form .search-btn {
   border-radius: .25rem!important;
+  max-height: 31px;
+  align-items: center;
```

Comment:
Why do we need this? I don't see it having any effect.

Reply:
With align-items:
<img width="176" alt="Screenshot 2024-08-30 at 10 37 58" src="https://github.com/user-attachments/assets/5133c052-1c99-46cf-944c-79a8878264c4">


Without (text at base of btn):
<img width="176" alt="Screenshot 2024-08-30 at 10 38 01" src="https://github.com/user-attachments/assets/05535804-4023-4acf-a47d-9057f9d7f005">

Reply:
So this is due to the fight between padding and max-height.  I think we can eliminate this if we also remove max-height and just set padding and font size appropriately.

Reply:
Whoops. I can try this still...